cmd/zetatool/inbound_ballot/inbound.go

```diff
@@ -0,0 +1,114 @@
+package inbound_ballot
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/zeta-chain/node/cmd/zetatool/config"
+	"github.com/zeta-chain/node/pkg/chains"
+	zetacorerpc "github.com/zeta-chain/node/pkg/rpc"
+)
+
+func NewFetchInboundBallotCMD() *cobra.Command {
+	return &cobra.Command{
+		Use:   "get-ballot [inboundHash] [chainID]",
```

Comment:
Instead of using config, would be simpler to just directly pass the argument here. For EVM and most chain, you just need a RPC? Command would be simpler to use as
```
get-ballot hash chain_id rpc
```
rather than setting up a config we can also set a default public rpc used automatically for each chain

Reply:
We have a default value for now for each chain; the config is not mandatory.
I think we should just make sure the default values are working, having just two parameters is optimal . Specially if we want to use a HTTP server in the future instead of the CLI

Reply:
Ok, make sense, I think it would still better having flags instead of a config.
```
get-ballot hash chain_id --rpc "..."

get-ballot hash chain_id --rpc "..." --bitcoin-username "...."
```

Reply:
I am not sure if that is a good idea, 
For example for a getting a btc ballot we would need flags for 
- User 
- Password 
- Params 
- Host 
- ZetachainRPC 
- ZetachainID ( Good to have ) 

Which is a minimum of 5 flags. Imo, it's just better to use a rather use a config file , which can be created easily using the sample or a shell scripts.

The config also does not need all values to be present, just the rpc that you want to use

Reply:
Bitcoin is very specific because requires your own node with auth, most if not all other chains will just require a rpc so I wouldn't make the system more complex for just one edge case. We can also have config specifically for Bitcoin. But having the ability to provide the rpc directly in the command line for most chians would make the solution simpler

Reply:
It would still need three inputs, `rpc` , `zetachainRpc` and `zetachainId` 

I think it's just cleaner to have the option of a config rather that having 3 flags , and a config (for bitcoin only ) 

Also, we should ideally just focus on having the correct rpcs listed in the default file directly, and not really provide any configuration at all , the config is primarily for unexpected situations. 

With that said , I can make the change to using the flags if its still needed

Reply:
Ok, let's try this way for now